### PR TITLE
#44 Added  a unique constraint for the backend URL

### PR DESCRIPTION
**Pull request by NJavokhir**
### Issue:
The backend URL field in the tunnel API currently allows multiple clients, leading to potential data mix-up issues due to shared JWT tokens.

### Solution:
Added a unique constraint for the backend URL field in the create client form to ensure each client has a distinct backend URL.

### Changes Made:
- Implemented unique constraint for backend URL field.
- Conducted manual and automated testing.
- Resolved conflicts.
- Tagged code owner and a second developer for review.

### Testing:
Manual testing confirms proper functionality of unique constraint.

### Note:
I'll handle branch merging and deletion. Open to feedback for improvement.

### CODE REVIEW

File: server/assets-v1/templates/src/pages/client_portal/register.html [hunk 1]
```diff
@@ -160,18 +163,19 @@ <h1 class="font-bold text-3xl md:text-4xl text-[#3751FE] mb-11 animate-slideFrom
...
-        id: "",
-        secret: "",
-        name: "",
-        redirect_uri: "",
-        backend_uri: "",
-      });
+      const showToast = ref(false);
+      const toastMessage = ref("");
+      const backend_uri_text = ref("");
+      const responseFromBackend = ref(false);
```
Comment (ternakkode) [inline server/assets-v1/templates/src/pages/client_portal/register.html:169]: lets change `responseFromBackend` to improve readability, something like `isBackendUrlUsed`

File: server/assets-v1/templates/src/pages/client_portal/register.html [hunk 2]
```diff
@@ -160,18 +163,19 @@ <h1 class="font-bold text-3xl md:text-4xl text-[#3751FE] mb-11 animate-slideFrom
...
-      const client = ref({
-        id: "",
-        secret: "",
-        name: "",
-        redirect_uri: "",
-        backend_uri: "",
-      });
+      const showToast = ref(false);
+      const toastMessage = ref("");
+      const backend_uri_text = ref("");
```
Comment (ternakkode) [inline server/assets-v1/templates/src/pages/client_portal/register.html:168]: incosistency, other variable is using camelCase but this variable is using snake_case

**Review by ternakkode — CHANGES_REQUESTED** [review]
need few adjustment

**Comment by NJavokhir** [comment]
@ternakkode Can you review this PR again.

**Review by ternakkode — APPROVED** [review]

**Pull request merged**